pkg/apis/rook.io/v1/types.go

```diff
@@ -161,6 +161,15 @@ type LabelsSpec map[KeyType]Labels
 // Labels are label for a given daemons
 type Labels map[string]string
 
+type GrowthPolicy struct {
+	// maxsize for the capacity of DeviceStorage
+	// Optional
+	Maxsize string `json:"maxsize,omitempty"`
+	//growthRate for percentage increment of claimed volume
+	// Optional
+	GrowthRatePercent int `json:"growthRatePercent,omitempty"`
```

Comment:
Godoc comment from above.
Descriptions should provide fuller instructions for users. It's not clear to me how to use this field.

```suggestion
	//GrowthRatePercent <fuller description of usage>
	// Optional
	GrowthRatePercent int `json:"growthRatePercent,omitempty"`
```